docs/source/examples/mg-mode-part-i.rst

```diff
@@ -0,0 +1,13 @@
+.. _notebook_mg_mode_part_i:
+
+=====================================
+Multi-Group Mode Part I: Introduction
```

Comment:
I like your explicit and clear naming convention - could you do something similar and rename the M(D)GXS notebook links as "Multi-Group (Delayed) Cross Section Generation: Part XXX"?